Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -398,6 +398,28 @@
         tags:
           - NukeOpsUplink
 
+- type: listing
+  id: UplinkDnaScramblerImplant
+  name: uplink-dna-scrambler-implanter-name
+  description: uplink-dna-scrambler-implanter-desc
+  icon: { sprite: /Textures/Mobs/Species/Human/parts.rsi, state: full }
+  productEntity: DnaScramblerImplanter
+  cost:
+    Telecrystal: 10
+  categories:
+    - UplinkImplants
+
+- type: listing
+  id: UplinkEmpImplanter
+  name: uplink-emp-implanter-name
+  description: uplink-emp-implanter-desc
+  icon: { sprite: /Textures/Objects/Magic/magicactions.rsi, state: shield }
+  productEntity: EmpImplanter
+  cost:
+    Telecrystal: 7
```

```suggestion
    Telecrystal: 5
```

With the EMP being limited to 3 uses, 10 seconds apart, lowering this down to 5TC wouldn't be an issue. Compared to SS13 it's a huge nerf since those were able to be used instantly AND also came with a box of EMP grnades all for 2TC.

---

so true

---

I lowered the cost to 5 TC, the original cost was based on EMP grenades so I lowered those from 4 to 3 TC. Probably fits in the scope of this PR.